examples/kubernetes/samba-ad-server-deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: samba-ad-server
+  labels:
+    app: samba
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: samba-ad
+  template:
+    metadata:
+      labels:
+        app: samba-ad
+    spec:
+      containers:
+      - name: samba
+        image: quay.io/samba.org/samba-ad-server:latest
+        securityContext:
+          capabilities:
+            add: ["SYS_ADMIN"]
+        ports:
+        - containerPort: 445
```

Comment:
As this is an ad server I think that at the least we should also be exposing ports for dns, ldap and perhaps ldaps.

Reply:
true, this was just a start. probably I should mark this Draft... ;-)

Reply:
updated with a TODO comment for now

Reply:
Heh. I think this is fine for now. Some of these might be considered "obsolete" but I don't know what applications and platforms use what. So I think this is good enough.

Reply:
Yeah, we can still clean up later.